test/controllers/comments_controller_test.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class CommentsControllerTest < ActionDispatch::IntegrationTest
+  test "コメントの作成に成功した場合、法案詳細ページにリダイレクトされる" do
+    bill = bills(:one)
+    assert_difference "bill.comments.count" do
```

Comment:
[MUST] すみません、もう1点追加で
assert_difference (assert_no_difference) や assert_changes (assert_no_changes) なんですが、 文字列以外にも call をメソッドとして持つオブジェクトも渡せます
文字列で書くよりも lambda で書いたほうがエディタでハイライトが効いてよみやすいと思うので、ここは lambda で書いちゃいましょう

```suggestion
    assert_difference -> { bill.comments.count } do
```

assert_difference (assert_no_difference) や assert_changes (assert_no_changes) のコード、メタプロを読む練習にもなりそうなので読んでみると面白いかもです〜😆